pkgs/by-name/ku/kubectl-ai/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "kubectl-ai";
+  version = "0.0.9";
+
+  src = fetchFromGitHub {
+    owner = "GoogleCloudPlatform";
+    repo = "kubectl-ai";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-zsqHRXUyRguDbY3yPCWJ9KLzF6VR/y7juzI9uaVPSAE=";
+  };
+
+  vendorHash = "sha256-VpTAq9rXDPBOdaZIXm7h4AP1YFOk7HzLiUxYaCOP/w8=";
+
+  # Build the main command
+  subPackages = [ "cmd" ];
+
+  postInstall = ''
+    mv $out/bin/{cmd,kubectl-ai}
+  '';
+
+  ldflags = [
+    "-s"
+    "-w"
```

Currently `kubectl-ai version` works abnormaly:

```
$ kubectl-ai version
version: dev
commit: none
date: unknown
```

```suggestion
    "-w"
    "-X main.version=${version}"
    "-X main.commit=${src.rev}"
    "-X main.date=unknown"
```